clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1851,8 +1851,8 @@ mlir::LogicalResult CIRToLLVMVAArgOpLowering::matchAndRewrite(
   return op.emitError("cir.vaarg lowering is NYI");
 }
 
-  /// Returns the name used for the linkage attribute. This *must* correspond
-  /// to the name of the attribute in ODS.
+/// Returns the name used for the linkage attribute. This *must* correspond
```

Comment:
This is an unrelated formatting change. It's pretty minor here, but it might be worth figuring out how to avoid them in general.

Reply:
ah, thank you for good catch. This is super interesting as I used 
git-clang-format --diff after I staged the files before I committed and pushed. Looks like even git-clang-format  introduce formatting change that is not really related to diff?

Reply:
@ghehg I mentioned this already to you but still useful: there are IDE setups to make the formatting only touch the lines you are modifying, you might be more successful with such setup looking forward.

Reply:
> @ghehg I mentioned this already to you but still useful: there are IDE setups to make the formatting only touch the lines you are modifying, you might be more successful with such setup looking forward.
This time is not about IDE, it's git-clang-format who introduced extra formatting. I figured out the reason, and shared it with @smeenai in our private chat. Worth sharing here too.
`kindaof figured out why git-clang-format gave me extra formatting. Its Version issue
I need to set 
git config --global clangFormat.binary $CLANGIR_RELEASE_PATH/bin/clang-format

without this, git-clang-format uses clang-format on $PATH which 
[)]$ clang-format --version
clang-format version 18.1.3`

Obviously, clang-format v20 has different formatting specs, But still cannot explain why git-clang-format would touch lines that’re not in my diff. 
It looks like git-clang-format tries its best to not affect lines you didn't touch but the heuristics don't work 100%